lua/rustaceanvim/config/internal.lua

```diff
@@ -265,6 +265,38 @@ local RustaceanDefaultConfig = {
     load_rust_types = function()
       return should_enable_dap_config_value(RustaceanConfig.dap.adapter)
     end,
+    --- @type DapClientConfig | disable | fun():(DapClientConfig | disable)
+    configuration = function()
+      local ok, _ = pcall(require, 'dap')
+      if not ok then
+        return false
+      end
+
+      -- default
+      local dap_config = {
+        name = 'Rust tools debug',
+        type = 'lldb',
+        request = 'launch',
+        stopOnEntry = false,
+      }
+
+      ---@diagnostic disable-next-line: different-requires
+      local dap = require('dap')
+      -- Load configurations from a `launch.json`.
+      -- It is necessary to check for changes in the `dap.configurations` table, as
+      -- `load_launchjs` does not return anything, it loads directly into `dap.configurations`.
+      local pre_launch = vim.deepcopy(dap.configurations) or {}
+      require('dap.ext.vscode').load_launchjs()
```

Comment:
I chose to load `launch.json` in the `configuration` field instead of in `dap.lua` because imo it is more transparent, and gives the user full control.

This can be easily moved if required.